frontend/internal-packages/db/schema/schema.sql

```diff
@@ -1120,6 +1133,144 @@ CREATE TABLE IF NOT EXISTS "public"."building_schemas" (
+    "thread_id" "text" NOT NULL,
+    "checkpoint_ns" "text" DEFAULT ''::"text" NOT NULL,
+    "channel" "text" NOT NULL,
+    "version" "text" NOT NULL,
+    "type" "text" NOT NULL,
+    "blob" "bytea",
+    "organization_id" "uuid" NOT NULL,
+    "created_at" timestamp with time zone DEFAULT "now"()
+);
+
+
+ALTER TABLE "public"."checkpoint_blobs" OWNER TO "postgres";
+
+
+COMMENT ON TABLE "public"."checkpoint_blobs" IS 'Stores channel values (state data) for checkpoints';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_blobs"."channel" IS 'Name of the channel containing state data';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_blobs"."version" IS 'Version number for channel value tracking';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_blobs"."type" IS 'Type hint for deserialization';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_blobs"."blob" IS 'Binary serialized data';
+
+
+
+CREATE TABLE IF NOT EXISTS "public"."checkpoint_migrations" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "v" integer NOT NULL,
+    "organization_id" "uuid" NOT NULL,
+    "created_at" timestamp with time zone DEFAULT "now"()
+);
+
+
+ALTER TABLE "public"."checkpoint_migrations" OWNER TO "postgres";
+
+
+COMMENT ON TABLE "public"."checkpoint_migrations" IS 'Tracks applied checkpoint system migrations';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_migrations"."v" IS 'Migration version number';
+
+
+
+CREATE TABLE IF NOT EXISTS "public"."checkpoint_writes" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "thread_id" "text" NOT NULL,
+    "checkpoint_ns" "text" DEFAULT ''::"text" NOT NULL,
+    "checkpoint_id" "text" NOT NULL,
+    "task_id" "text" NOT NULL,
+    "idx" integer NOT NULL,
+    "channel" "text" NOT NULL,
+    "type" "text",
+    "blob" "bytea" NOT NULL,
+    "organization_id" "uuid" NOT NULL,
+    "created_at" timestamp with time zone DEFAULT "now"()
+);
+
+
+ALTER TABLE "public"."checkpoint_writes" OWNER TO "postgres";
+
+
+COMMENT ON TABLE "public"."checkpoint_writes" IS 'Stores pending write operations for checkpoints';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_writes"."task_id" IS 'Identifier of the task that generated this write';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_writes"."idx" IS 'Index for ordering multiple writes from the same task';
+
+
+
+COMMENT ON COLUMN "public"."checkpoint_writes"."channel" IS 'Target channel for the write operation';
+
+
+
+CREATE TABLE IF NOT EXISTS "public"."checkpoints" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "thread_id" "text" NOT NULL,
+    "checkpoint_ns" "text" DEFAULT ''::"text" NOT NULL,
+    "checkpoint_id" "text" NOT NULL,
+    "parent_checkpoint_id" "text",
+    "checkpoint" "jsonb" NOT NULL,
+    "metadata" "jsonb" DEFAULT '{}'::"jsonb" NOT NULL,
+    "organization_id" "uuid" NOT NULL,
+    "created_at" timestamp with time zone DEFAULT "now"(),
+    "updated_at" timestamp with time zone DEFAULT "now"()
```

Comment:
[ask]
I think only the `checkpoints` table has `updated_at`, is there a case where the record is updated?👀